docs/usage.md

```diff
@@ -101,14 +101,14 @@ In this case, there is no `match` key, so only one rule needs to match:
 `rules` is an array of rules that can be used to target specific files with your defined syntax file.  The rules are processed until the first rule matches, so order your rules in a way that makes sense to you.
 
 ### Filename Rule
-Filename rule defines a filename with regex.  This uses the full file path.
+Filename rule defines a filename with regex.  The pattern is matched against the beginning of the full file path (there is an implicit `^`).
```

Comment:
I wouldn't say it is always implicit, as you can define it as one rule does in the settings file `{"file_name": "^.*(\\.jshintrc|\\.jscsrc|\\.csslintrc)$"}`.  The nature of the regex being applied requires the full string to match.  You could probably drop off the `$` and still have it work as well, but for some reason people always define the `$` and not the `^`.  But the real point is the regex is applied to the **whole** path no just file name....I know, I am not the original author of the plugin, so I do carry around some of the legacy baggage; I would have called it `file_path`.